lib/cherry_silt/connection.rb

```diff
@@ -56,24 +52,29 @@ def receive_data(data)
   end
 
   def unbind
+    puts 'Bye Bye'
+    # this doesn't seem to be dropping connection
     @server.connections.delete(self)
   end
 
   def fetch_user(name)
-    @user_data = @mongo_client[:users].find(name: name).first
-    @user_data = {name: 'unknown_user', password: 1} if @user_data.nil?
+    p = @client[:player].find(name: name).first
+    begin
+      @player = CherrySilt::Player.from_h p
+    rescue
+      @player = CherrySilt::Player.new 'unknown_user' if @player.nil?
```

Comment:
Is 'unknown_user' a placeholder for now?  If the name is not found in the rescue block we should probably move to a character creation method(s).